packages/core/src/components/table/table-footer/table-footer.tsx

```diff
@@ -32,6 +32,10 @@ export class TdsTableFooter {
   /** Sets the pagination number. */
   @Prop({ reflect: true, mutable: true }) paginationValue: number = 1;
 
+  @Prop({ reflect: true}) rowsperpage: boolean = true;
```

Comment:
```suggestion
  @Prop({ reflect: true}) rowsPerpage: boolean = true;
```

Reply:
The prop is not camelCase to avoid passing prop as `rows-per-page` in html